modules/lib/src/main/resources/assets/js/app/inputtype/ui/selector/image/ImageEditor.ts

```diff
@@ -718,20 +723,25 @@ export class ImageEditor
         let editContainer = new DivEl('edit-container');
 
         this.editResetButton = new Button(i18n('button.reset'));
-        this.editResetButton.setVisible(false).addClass('transparent').onClicked((event: MouseEvent) => {
-            event.stopPropagation();
+        this.editResetButton
+            .addClass('reset-button')
+            .addClass('transparent')
+            .setVisible(false)
+            .onClicked((event: MouseEvent) => {
+                event.stopPropagation();
 
-            if (this.isFocusEditMode()) {
-                this.resetFocusPosition();
-                this.resetFocusRadius();
-            } else if (this.isCropEditMode()) {
-                this.resetZoomPosition();
-                this.resetCropPosition();
-            }
-        });
+                if (this.isFocusEditMode()) {
+                    this.resetFocusPosition();
+                    this.resetFocusRadius();
+                } else if (this.isCropEditMode()) {
+                    this.resetZoomPosition();
+                    this.resetCropPosition();
+                }
+            });
 
-        let applyButton = new Button(i18n('button.apply'));
-        applyButton.addClass('blue').onClicked((event: MouseEvent) => {
+        const applyButton = new Button(i18n('button.apply'));
+        applyButton.addClass('apply-button').addClass('blue');
```

Comment:
You can set multiple classes in one go: 

`applyButton.addClass('apply-button blue');`